attributed_text/src/lib.rs

```diff
@@ -0,0 +1,19 @@
+// Copyright 2025 the Parley Authors
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+
+//! Attributed Text is a Rust crate which ...
+//!
+//! ## Features
+//!
+//! - `std` (enabled by default): Use the Rust standard library.
```

Comment:
Features should document why you'd use them, not just "how" they work. Can we just add something like: "this is currently unused and is provided for forward compatibility".

Reply:
Well for example [`color`](https://docs.rs/color/latest/color/#features) says why you would enable the feature.